docs/usage/examples/opentelemetry.md

```diff
@@ -72,12 +93,23 @@ service:
   pipelines:
     traces:
       receivers: [otlp]
-      exporters: [otlp/jaeger, logging]
-      processors: [spanmetrics, batch]
+      exporters:
+        - otlp/jaeger
+        # Send traces to connector for metrics calculation
+        - spanmetrics
+        # Enable debug exporter to see traces in the logs
+        #- debug
+      processors: [batch]
 
     metrics:
-      receivers: [otlp]
-      exporters: [prometheus]
+      receivers:
+        - otlp # allow to receive metrics directly from Renovate
+        - spanmetrics # allow receiving metrics calculated internally by the spanmetrics connector
+      processors: [batch]
+      exporters:
+        - otlphttp/prometheus
+        # Enable debug exporter to see metrics in the logs
+        # - debug
```

Comment:
The `- debug` bit is commented out here again.

Reply:
For user debugging